test/test_allocation_domain.cpp

```diff
@@ -1299,4 +1299,13 @@ TEST_F(AllocationDomainTest, Issue1290_ReplayCasPFailedDueToDifferentRanks) {
   EXPECT_THAT(out_tensor.sizes(), ElementsAre(2));
 }
 
+TEST_F(AllocationDomainTest, TrivialStrideOrderTensorViewBuilder) {
```

Comment:
JFYI, this is testing implementation not behavior, which isn't ideal. We should instead create a FusionExecutorCache level test that verifies things have been vectorized properly. That being said, currently, having this test as is is still better than not having any test, so LGTM!

Reply:
That's understood.

I was tracking the vectorization issue here: [1399](https://github.com/NVIDIA/Fuser/issues/1399).
I'll put test verification for vectorization there when I have something concrete.